core/src/main/java/bisq/core/btc/wallet/TradeWalletService.java

```diff
@@ -1216,4 +1219,25 @@ private void applyLockTime(long lockTime, Transaction tx) {
         tx.getInputs().forEach(input -> input.setSequenceNumber(TransactionInput.NO_SEQUENCE - 1));
         tx.setLockTime(lockTime);
     }
+
+    private boolean removeDust(Transaction tx) {
+        List<TransactionOutput> txos = tx.getOutputs();
+        List<TransactionOutput> toKeep = new ArrayList<TransactionOutput>();
+        for (TransactionOutput zz: txos) {
+            if (zz.getValue().value >= 546) {
+                toKeep.add(zz);
+            }
+            else {
+                log.info("removing dust TXO = {}", zz.getValue().toFriendlyString());
```

Comment:
A more verbose log will help log readers.

Reply:
Thankyou for the review!  I have made the changes you suggested.  Here is an example of the more verbose logging as a result of this change:

![image](https://user-images.githubusercontent.com/47253594/77716425-2be97680-6fac-11ea-817b-81b3cc4b6fe2.png)